Tradução da página 'projetos-relacionados'. (#108)
okfn-brasil/querido-diario-comunidade: docs/en/source/understanding/related-projects.rst

Related Projects
###########################

> With Querido Diário handling the heavy and innovative work of collecting, storing, and making thousands of municipal gazettes available, get to know projects that already use our data.

Reviewer: ```suggestion
With Querido Diário handling the heavy and innovative work of collecting, storing and making thousands of municipalities gazettes available, get to know projects that already use our data.
```

Usamos "municipalities" na outra página, ne? vamos manter o padrão. Podemos sempre assumir que decisões de tradução que já adotamos em outras páginas vão valer pras seguintes também

`Querido Diário\: Technologies in Education`_
************************************************

> `Querido Diário\: Technologies in Education`_ is capable of indentifying official acts related to the world of technologies acquired by the basic education network at the municipality level in specific themes such as accessibility, connectivity, management, infrastructure,  artificial inteligence, robotics, etc.

Reviewer: Só corrigindo alguns typos e também retirei o "the world of technologies" pq acho que é uma figura de linguagem ("universo das tecnologias") que funciona em português, mas não tanto em Inglês.

> `Querido Diário\: Technologies in Education`_ is capable of indentifying official acts related to the world of technologies acquired by the basic education network at the municipality level in specific themes such as accessibility, connectivity, management, infrastructure,  artificial inteligence, robotics, etc.

Reviewer: Só corrigindo alguns typos e também retirei o "the world of" pq acho que é uma figura de linguagem ("universo das tecnologias") que funciona em português, mas não tanto em Inglês.

```suggestion
`Querido Diário\: Technologies in Education`_ is capable of identifying official acts related to  technologies acquired by the basic education network at the municipality level in specific themes such as accessibility, connectivity, management, infrastructure, artificial intelligence, robotics, etc.
```

.. image:: https://querido-diario-static.nyc3.cdn.digitaloceanspaces.com/docs/related-projects/querido-diario-tecnologias-educacao.png
    :alt: "Querido Diário: Technology in education" Webpage
    
`Diário do Clima`_
**********************************

> `Diário do clima`_ monitors gazettes to identify the most relevant documents on public policies related to climate and the enviroment. Users can track official action related to environmental crimes, biodiversity, mining, water, sanitation, etc. as soon as new official acts are published.

Reviewer: ```suggestion
`Diário do Clima`_ monitors gazettes to identify the most relevant documents on public policies related to climate and the environment. Users can track official acts related to environmental crimes, biodiversity, mining, water, sanitation, etc. as soon as new official gazettes are published.
```

.. image:: https://querido-diario-static.nyc3.cdn.digitaloceanspaces.com/docs/related-projects/diario-do-clima.png
   :alt: "Diário do clima" webpage.
 
`LegislaTech`_
*****************
 

> `LegislaTech`_ monitors public documents - specially legislative proceedings, judicial documents, gazettes, and even news- seeking topics of interest for users.

Reviewer: Em português está "tramitações legislativas, judiciais" aqui querendo dizer tramitações legislativas e tramitações judiciais. 

```suggestion
`LegislaTech`_ monitors public documents - specially legislative and judicial proceedings, official gazettes and even news - seeking topics of interest for users.
```

 
.. image:: https://querido-diario-static.nyc3.cdn.digitaloceanspaces.com/docs/related-projects/legislatech.png
    :alt: LegislaTech webpage.
    

`Exoonero`_
*******************

> `Exoonero`_ classifies, among appointments and dismissals, the official acts from the gazettes of the Association of Alagoan Municipalities, providing general and specific statistcs by municipality and by year.

Reviewer: Mesma coisa da outra correção: essa foi a forma que adotamos na outra tradução e estamos seguindo o padrão aqui.

```suggestion
`Exoonero`_ classifies, among appointments and dismissals, the official acts from the Municipalities Association of Alagoas State Official Gazette, providing general and specific statistcs by municipality and by year.
```

Author: Eu acho que esqueci esses pequenos detalhes, peço desculpas.... Vou tentar não repetir na próxima

.. image:: https://querido-diario-static.nyc3.cdn.digitaloceanspaces.com/docs/related-projects/exoonero.png
    :alt: Exoonero Webpage.

> Ro-Dou

Reviewer: ```suggestion
Ro-DOU
```
DOU aqui é uma sigla pra Diario Oficial da União, e como é o nome do projeto, fica tudo maiúsculo

*************
 

> Ro-Dou is a tool of the Brazilian Ministry of Economy that supports the daily activities of various public agencies by generating email *clippings* for officials. Initially create to monitor the Official Gazette of the Union, it has also begun using municipal gazettes as a source.

Reviewer: "Officials" não está errado, mas como já estamos usando muito pra falar de Diário (Officials Gazettes), bora usar um sinônimo para ficar menos confuso. 

E o Union Official Gazettes foi mudado também para ficar igual ao que fizemos na outra tradução.

```suggestion
Ro-DOU is a tool of the Brazilian Ministry of Economy that supports the daily activities of various public agencies by generating email *clippings* for public servants. Initially create to monitor the Union Official Gazette, it has also begun using municipalities gazettes as a data source.
```

> **Learn more**: `Ro-DOU Registration in the Ministry's Solutions Catalog`_.

Reviewer: Como a documentação em inglês é voltada para leitores internacionais, quando estamos referenciando páginas em português é bom adicionar um aviso que está em PT-BR. 

```suggestion
**Learn more**: `Ro-DOU Registration in the Ministry's Solutions Catalog (in PT-BR)`_.
```

Você não chegou nessa página ainda, mas vai ver que na página de raspadores, fazemos o mesmo pra avisar que o texto está em ingles: 
![image](https://github.com/user-attachments/assets/757be060-b6f4-4c67-9bc8-36ed11b4ccc8)

Author: Entendido.... Realmente eu não me atentei para isso anteriormente....

 
.. REFERÊNCIAS
.. _Querido Diário\: Technologies in Education: https://queridodiario.ok.org.br/educacao
.. _Diário do Clima: https://diariodoclima.org.br/
.. _LegislaTech: https://legisla.tech/
.. _Exoonero: https://exoonero.org/

> .. _Ro-DOU Registration in the Ministry's Solutions Catalog: https://www.gov.br/economia/pt-br/acesso-a-informacao/acoes-e-programas/transformagov/catalogodesolucoes/ro-dou

Reviewer: ```suggestion
.. _Ro-DOU Registration in the Ministry's Solutions Catalog (in PT-BR): https://www.gov.br/economia/pt-br/acesso-a-informacao/acoes-e-programas/transformagov/catalogodesolucoes/ro-dou
```
